test/unit/web-worker/test-amp-worker.js

```diff
@@ -57,7 +57,7 @@ describe('invokeWebWorker', () => {
       .callsFake(() =>
         Promise.resolve({
           text() {
-            return Promise.resolve();
+            return Promise.resolve('');
```

Comment:
If you include some `//# sourceMappingURL=foo.js` here, we should be able to test this.